scripts/py_matter_yamltests/matter_yamltests/parser_config.py

```diff
@@ -0,0 +1,27 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the 'License');
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an 'AS IS' BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from .yaml_loader import YamlLoader
+
+
+class TestConfigParser:
```

Comment:
Adding doc string everywhere would really help reviewing for this really large change

Reply:
I have added various docstrings over the files. Let me know if it helps.